<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainProjectWidget</class>
 <widget class="QWidget" name="MainProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projeto</string>
  </property>
  <widget class="QPushButton" name="btn_choose_video">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Escolha o Vídeo</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/folder_open.png</normaloff>:/i/folder_open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_main_video">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_annotation">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>610</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar Anotação</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/comment_add.png</normaloff>:/i/comment_add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame_notes">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>301</width>
     <height>591</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anotações:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_notes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_generate_ncl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>610</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar...</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/notepad.png</normaloff>:/i/notepad.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_project">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>610</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar Projeto</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/save.png</normaloff>:/i/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>671</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="player_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>691</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame_edit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>600</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_ok">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/i/check.png</normaloff>:/i/check.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/i/no.png</normaloff>:/i/no.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_save_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar alterações?</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_close_project">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>610</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fechar Projeto</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/folder.png</normaloff>:/i/folder.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
